domain/sysvar_cache.go

```diff
@@ -201,3 +204,9 @@ func (do *Domain) SetPDClientDynamicOption(option pd.DynamicOption, val interfac
 	}
 	return pdClient.UpdateOption(option, val)
 }
+
+// SetStatsCacheCapacity sets statsCache cap
+func (do *Domain) SetStatsCacheCapacity(c int64) {
+	do.StatsHandle().SetStatsCacheCapacity(c)
+	logutil.BgLogger().Info("update stats cache capacity successfully", zap.Int64("capacity", c))
```

Comment:
@Yisaer can we change this to debug? I just rebuilt from master and its a little noisy. It re-runs whenever sysvar cache is rebuilt (every 30s).